tests/test_schema.py

```diff
@@ -186,12 +184,27 @@ def test_raises_usetypeinvalid(type_):
         assert schema.UseType(type_) == type_
 
 
+def test_takedown():
+    """Test that a valid takedown passes validation."""
+    full_doc = load_json('valid.json')
+    full_doc['action'] = 'takedown'
+    actual_full_doc = schema.validate_schema(schema.takedown, full_doc)
+    assert actual_full_doc == full_doc
+
+
 def test_valid():
     """Test a valid document."""
     doc = load_json('valid.json')
     assert schema.track_bundle(doc) == doc
 
 
+def test_upsert():
+    """Test the upsert schema."""
+    upsert = load_json('valid.json')
+    takedown = copy.deepcopy(upsert)
+    assert schema.validate_schema(schema.delivery, upsert) == upsert
```

Comment:
In the other tests you capture the output of `schema.validate_schema` into variable and compare that to the original value. You should probably follow that same pattern here.